frontend/src/components/cluster/Chooser.tsx

```diff
@@ -38,8 +38,8 @@ export interface ClusterTitleProps {
 }
 
 export function ClusterTitle(props: ClusterTitleProps) {
-  const cluster = props.cluster || useCluster();
-  const clusters = props.clusters || useClustersConf();
+  const cluster = props.cluster;
```

Comment:
The useCluster/useClusterConf could be called, and then later:

```ts
const clusterUse = useCluster();
const clustersUse = useClustersConf()
...

const cluster = props.cluster || clusterUse;
const clusters = props.clusters || clustersUse;
```

This way the props could still be used to override.

Reply:
I think the way to do this is to use just the props and then have two higher level components which take either props or nothing. That way we can have the hooks or props properly. (This is what I did for the resource table, BTW)

At this point, if it's not being overridden, I prefer not to have it available.

Reply:
Yeah, just taking props is better.